`azurerm_linux[windows]_function[web]_app_[slot]` `azurerm_flex_consumption_app` - update api version to `2025-05-01` (#32239)
terraform-providers/terraform-provider-azurerm: website/docs/r/linux_function_app.html.markdown

~> **Note:** The feature can also be enabled via the app setting `WEBSITE_PULL_IMAGE_OVER_VNET`. Must be set to `true` when running in an App Service Environment.

> * `virtual_network_application_traffic_enabled` - (Optional) Should the traffic for the application be routed over virtual network enabled. Defaults to `false`.

Reviewer: Could we clarify this description? The phrase `routed over virtual network enabled` is difficult to parse.

```suggestion
* `virtual_network_application_traffic_enabled` - (Optional) Whether application traffic is routed through the virtual network. Defaults to `false`.
```

**Reference:** [reference-documentation-standards (descriptions)](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/reference-documentation-standards.md#descriptions)

* `vnet_route_all_enabled` - (Optional) Should all outbound traffic to have NAT Gateways, Network Security Groups and User Defined Routes applied? Defaults to `false`.

> ~> **Note:** In the `2023-12-01` API, `vnet_route_all_enabled` only controls application traffic. It is deprecated in favour of the `vnet_application_traffic_enabled` field in v5.0 provider.

Reviewer: This page is still documenting `vnet_route_all_enabled` even though the schema now marks it as legacy-only behind the `FivePointOh` compatibility path. I think this should be removed from the current reference docs so the docs only describe the vNext surface area.

Reference: [guide-breaking-changes.md](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/guide-breaking-changes.md#breaking-schema-changes-and-deprecations)
